sdk/python/tests/compiler/compiler_withitems_test.py

```diff
@@ -0,0 +1,94 @@
+from kfp.compiler import compiler
+import kfp.dsl as dsl
+
+
+# reproducible UUIDs: https://stackoverflow.com/a/56757552/9357327
+import uuid
+import random
+# -------------------------------------------
+# Remove this block to generate different
+# UUIDs everytime you run this code.
+# This block should be right below the uuid
+# import.
+rd = random.Random()
+rd.seed(0)
+uuid.uuid4 = lambda: uuid.UUID(int=rd.getrandbits(128))
+# -------------------------------------------
+
+
+# @dsl.pipeline(name='my-pipeline')
+# def pipeline(my_pipe_param=10):
```

Comment:
Is this test failing?

Reply:
shouldn't be

Reply:
Should there be commented out code here?